public/admin/config.yml

```diff
@@ -726,3 +726,59 @@ collections:
                 name: vaccinationSignupUrl
                 required: false
                 widget: string
+  - label: Vaccination Phases & Alerts
+    name: vaccinationPhasesAndAlerts
+    label_singular: Vaccination Phase
+    description: Details on vaccination phases and alerts.
+    format: json
+    files:
+      - name: Vaccine Information-State
+        label: State Vaccine Information 
+        file: src/cms-content/vaccines/state-vaccine-phases.json
+        fields:
+          - label: Regions
+            name: regions
+            widget: list
+            fields:
+              - label: Location Name
+                name: locationName
+                widget: string
+              - label: Fips Code
+                name: fips
+                widget: string
+              - label: Hidden
+                name: hidden
+                widget: boolean
+                required: false
+              - label: Eligibility Information URL
+                name: eligibilityInfoUrl
+                widget: string
+              - label: Email alert version
+                name: emailAlertVersion
+                default: 0
+                widget: number
+              - label: Phases
+                name: eligibilityPhases
+                widget: list
+                collapsed: true
+                fields:
+                  - label: Group Phase
+                    name: groupPhase
+                    widget: string
+                  - label: Group Tier
+                    name: groupTier
+                    widget: string
+                  - label: Group Description
+                    name: groupDescription
+                    widget: text
+                  - label: Expanded Definition URL
+                    name: expandedDefinitionUrl
+                    required: false
+                    widget: string
+                  - label: Eligibility Start Date
+                    name: eligibilityStartDate
+                    widget: date
```

Comment:
Most future phases don't have a defined date, or have it in a guesstimate form "May-June" instead of firm dates, consider changing the widget type to text. An alternative could be to keep it as date, and render as "to be determined" if the date is not set, or to always render as Month only if the data is in the future

Reply:
Yeah I was treating this as a day that it actually starts - so if it wasn't a specific date don't use it, but maybe that's right that it should be text.  can always start there